### ISSUE TITLE
Provide instances for Duration

### ISSUE THREAD

**Issue by i-am-the-slime** [rowtype-yoga/purescript-yoga-json#7]
It is very nice to be able to type a random `number` in some `JSON` with its correct temporal granularity.

`{ secondsSinceLaunch :: Seconds }`

Therefore I want straightforward newytpe derived instances.

**Comment by jgarte** [comment]
What files would I have to modify to implement this? Would I need to use the `purescript-datetime` library?

**Comment by i-am-the-slime** [comment]
@jgarte Yes, you would add a dependency on the `purescript-datetime` library by running:

```
spago install datetime
```

Then you would add an instance in [JSON.purs](https://github.com/rowtype-yoga/purescript-yoga-json/tree/main/src/Yoga/JSON.purs)

and add a test in [BasicsSpec.purs](https://github.com/rowtype-yoga/purescript-yoga-json/tree/main/test/BasicsSpec.purs)

I'm not sure but maybe it's even possible to use `derive newtype instance` to generate the instances.

**Comment by i-am-the-slime** [comment]
Closed via #17
